microsoft-edge/webview2/concepts/browser-features.md

```diff
@@ -153,7 +153,7 @@ The following shortcuts are always turned off in WebView2, or are effectively tu
 | Browser Task Manager `*` | **Shift+Escape** |
 | Edge Feedback `*` | **Shift+Alt+I** |
 | Mute Tab `*` | **Ctrl+M** |
-| New Incognito Window | **Ctrl+Shift+N** |
+| New Incognito<!-- todo: InPrivate? --> Window | **Ctrl+Shift+N** |
```

Comment:
```suggestion
| New InPrivate Window (sometimes called _incognito_) | **Ctrl+Shift+N** |
```
@sivMSFT - should we mention here:
* incognito
* Incognito
* InPrivate
* both

Reply:
result:
```
| New InPrivate (Incognito) Window | **Ctrl+Shift+N** |
```